proposal/Xi-Plus/README.rst

```diff
@@ -0,0 +1,78 @@
+==================================
+Proposal for shared-data-encrypter
+==================================
+
+Basic information
+=================
+
+GitHub repository: https://github.com/Xi-Plus/shared-data-encrypter
+
+An encryption library provides encrypted data that you want to send to multiple people. You don't need to send lots of different copies. And the recipients don't share the key.
```

Comment:
I am not sure cryptography involves real numbers.  If your applications don't, please make a good argument why it should use the techniques covered in the course.

Reply:
Cryptography algorithm usually invokes integers (e.g., RSA, AES). Some algorithm use matrix calculation in encryption processes.